src/network.c

```diff
@@ -1226,8 +1226,8 @@ static int parse_part_encr_aes256 (sockent_t *se, /* {{{ */
   /* Make sure at least the header if available. */
   if (buffer_len <= PART_ENCRYPTION_AES256_SIZE)
   {
-    NOTICE ("network plugin: parse_part_encr_aes256: "
-        "Discarding short packet.");
+    ERROR ("network plugin: Decryption failed: "
+           "Discarding short packet.");
```

Comment:
Please keep the previous indentation (two extra tabs in this case). Applies to all of your change.